src/pyxplor/pyxplor.py

```diff
@@ -0,0 +1,52 @@
+def time_series_explorer(input_df: pd.DataFrame, 
+                         date_column: str, 

```

Comment:
Following @phchen5 comment re: input being list of strings - change this line to reflect list type

Reply:
The input is a dataframe, so the column type should be str. We can also think about date type, but not sure the list of strings suggestion.

Reply:
I think what we're trying to say is that, if you're only taking in one string value, it means that we are only plotting one line graph. But what if we have a number of date/time variables in the dataset? We would want to visualize all of them right? So that's why we want to create subplots that take in each one of those time variables. So we feed into the function a series (a list) of time variables. Here's an example of an outcome.

![image](https://github.com/UBC-MDS/PyXplor/assets/69966518/e6812db0-28cf-454a-99d7-7abf8c36da8d)

Reply:
Ohhh I get it, you are right. Let me work on that.